src/event-manager.js

```diff
@@ -44,11 +44,33 @@ function preventDefault(evt) {
 // and gestural input (e.g. 'click', 'tap', 'panstart').
 // Delegates gesture related event registration and handling to Hammer.js.
 export default class EventManager {
-  constructor(element, options = {}) {
-    this.element = element;
+  constructor(element = null, options = {}) {
     this.options = options;
+    this.eventHandlers = [];
+
     this._onBasicInput = this._onBasicInput.bind(this);
+    this._onOtherEvent = this._onOtherEvent.bind(this);
```

Comment:
Not really related to this diff, but can we remove `bind` and define these methods as arrow functions instead?